lib/logstash/filters/elasticsearch/patch.rb

```diff
@@ -0,0 +1,26 @@
+# encoding: utf-8
+require "elasticsearch"
+require "elasticsearch/transport/transport/http/manticore"
+
+es_client_version = Gem.loaded_specs['elasticsearch-transport'].version
+if es_client_version >= Gem::Version.new('7.2') && es_client_version < Gem::Version.new('7.15')
```

Comment:
we're assuming elasticsearch client would resolve the issue in 7.15.0, otherwise this will need to be revisited